DolbyIO/Source/Public/DolbyIOParticipantInfo.h

```diff
@@ -4,6 +4,33 @@
 
 #include "DolbyIOParticipantInfo.generated.h"
 
+UENUM(BlueprintType)
+enum class EDolbyIOParticipantStatus : uint8
```

Comment:
Is there any reason for specifying which underlying type this enum is using?

Reply:
no (just the smallest) - any suggestions?

Reply:
I would just remove this specifier.

Reply:
unfortunately UE requires it 😜